src/fidesops/core/config.py

```diff
@@ -266,15 +302,15 @@ def get_config() -> FidesopsConfig:
     """
     try:
         return FidesopsConfig.parse_obj(load_toml("fidesops.toml"))
-    except (FileNotFoundError, ValidationError) as e:
```

Comment:
Catching the `ValidationError` here was odd; it basically made it try loading the config twice:
- once from the file, which if invalid threw an error
- then once again from defaults, which may throw an error

This is surprising, because if you have an invalid config it fails twice, and usually with different error messages, which is misleading.

Reply:
By removing this catch, the validation error simply crashes startup and throws a more relevant warning log and stacktrace.

Reply:
good catch, I'll actually have to update this on my fideslog integration branch

Reply:
Ideally let's get this merged to `main` so you can just rebase 👍